app/routes/events/components/ImageGallery.tsx

```diff
@@ -0,0 +1,20 @@
+import Modal from 'app/components/Modal';
+import type { imageGallery } from 'app/models';
+
+const ImageGalleryComponent = ({
+  imageGallery,
+}: {
+  ImageGallery: imageGallery;
+}) => {
+  return (
+    <>
+      <Modal>
```

Comment:
In the image you showed, the modal has a title ("Bildegalleri:"). I don't see where it is set. I was going to say that it didn't need the semicolon and that it didn't need to be centered, so that it matches the other modals.

Reply:
Ok somehow this is just one of the WIP commits, not sure how but the rebase got messed up... I'll try to push the latest one tomorrow probably